RESOURCES.md

```diff
@@ -74,7 +85,6 @@
 
 ### Selected Talks
 
-- _3D Tiles Next: Bringing Massive 3D Geospatial Data to the Web_, WebGL + WebGPU Meetup (April 2022). [Video and slides](https://cesium.com/learn/presentations/#webg-webgpu-meetup)
```

Comment:
We should keep this talk and the selected articles below. The branches must diverged at some point.